cmd/fastwalk.go

```diff
@@ -11,173 +11,12 @@ package cmd
 import (
```

Comment:
now that we don't do what fastwalk does, does it make sense to keep this code? @klauspost - perhaps move this to the new data-usage file? or rename this file as some `data-usage-utils.go` ?